modules/arrow/package.json

```diff
@@ -44,8 +44,9 @@
     "fs": false
   },
   "scripts": {
-    "pre-build": "npm run build-worker && npm run build-bundle && npm run build-bundle -- --env=dev && npm run build-triangulation-worker",
-    "build-bundle": "ocular-bundle ./src/index.ts",
+    "pre-build": "npm run build-worker && npm run build-bundle && npm run build-bundle-dev && npm run build-triangulation-worker",
```

Comment:
Should this be `prepublishOnly` everywhere? Or good to stick with `pre-build`? Efficient, but a bit uncanny that some builds only happen on publish.

Reply:
Some packages require pre-build to run (copy of libs, build workers etc.)